src/variable.sass

```diff
@@ -0,0 +1,2 @@
+$gradient-bg: linear-gradient(to bottom, #11998e, #38ef7d)
```

Comment:
I feel the bottom colour is too bright. Can we tone down the diff a little?

Reply:
Ok bro, let me adjust a little bit.